src/java.desktop/windows/classes/sun/awt/shell/Win32ShellFolderManager2.java

```diff
@@ -602,27 +588,22 @@ public void run() {
                     }
                 }
             };
-            comThread = AccessController.doPrivileged((PrivilegedAction<Thread>) () -> {
-                String name = "Swing-Shell";
-                 /* The thread must be a member of a thread group
-                  * which will not get GCed before VM exit.
-                  * Make its parent the top-level thread group.
-                  */
-                Thread thread = new Thread(
-                        ThreadGroupUtils.getRootThreadGroup(), comRun, name,
-                        0, false);
-                thread.setDaemon(true);
-                /* This is important, since this thread running at lower priority
-                   leads to memory consumption when listDrives() function is called
-                   repeatedly.
-                 */
-                thread.setPriority(Thread.MAX_PRIORITY);
-                return thread;
-            });
+            /* The thread must be a member of a thread group
```

Comment:
line 579 above (which does not show in the diff still has SuppressWarnings("removal")
I'm not sure why it is still needed. Oversight ?

Reply:
missed earlier..removed..